src/salsa.md

```diff
@@ -11,8 +11,14 @@ Matsakis.
 
 > As of <!-- date-check --> November 2022, although Salsa is inspired by
 > (among other things) rustc's query system, it is not used directly in rustc.
-> It _is_ used in chalk and extensively in `rust-analyzer`, but there are no
-> medium or long-term concrete plans to integrate it into the compiler.
+> It _is_ used in [chalk], the library for the trait system, and extensively in
```

Comment:
```suggestion
> It _is_ used in [chalk], an implementation of  Rust's trait system, and extensively in
```

or something like that. “The library” sounds like it's the only library and actively used. However, Chalk is actually no longer used in the compiler (it used to be available under `-Zchalk` and later `-Ztrait-solver=chalk` before it got ripped out).